build/Publish.targets

```diff
@@ -318,8 +318,8 @@
                     UploadTimeoutInMinutes="$(PushToBlobFeed_UploadTimeoutMinutes)"
                     ManifestBranch="$(BuildBranch)"
                     ManifestBuildId="$(Version)"
-                    ManifestBuildData="ProductVersion=$(PackageVersion);UniverseCommitHash=$(CommitHash)"
-                    ManifestCommit="$(CommitHash)"
+                    ManifestBuildData="ProductVersion=$(PackageVersion);UniverseCommitHash=$(RepositoryCommit)"
+                    ManifestCommit="$(RepositoryCommit)"
```

Comment:
@dotnet/dnceng  we use the `PushToBlobFeed` task from the Microsoft.DotNet.Build.Tasks.Feed v2.1.0-prerelease-02430-04 package. I see Arcade now recommends the `PushToAzureDevOpsArtifacts` task. Would that be appropriate in dotnet/aspnetcore's release/2.1 branch❔ We're not using any of the Arcade job or jobs templates here.

Somewhat separately, I believe `UniverseCommitHash` is not the metadata other repos set in their manifests. What's the correct name (for consistency)❔

Reply:
@riarenas Would you be able to help with answering this?

Reply:
>https://github.com/orgs/dotnet/teams/dnceng we use the PushToBlobFeed task from the Microsoft.DotNet.Build.Tasks.Feed v2.1.0-prerelease-02430-04 package. I see Arcade now recommends the PushToAzureDevOpsArtifacts task. Would that be appropriate in dotnet/aspnetcore's release/2.1 branch

Arcade was brought up during the 3.0 cycle, so we don't expect any repo to use those tasks for their 2.1 branches.

As for the metadata in the manifest, I'm not familiar enough with the 2.1 infrastructure to have a sense if there's even any consistency here. @mmitche maybe would be able to help with that?

Reply:
Relevant bit of build.xml manifest for 2.1.30 shows we're inconsistent but I'm not sure if something upstream is expecting `UniverseCommitHash` from us and `Commit` from everything else. @mmitche❔

``` xml
<Build Name="aspnet-extensions" BuildId="2.1.30-servicing-21412-01+pb-20210811-01" ProductVersion="2.1.30" Branch="internal/release/2.1" Commit="7af3eb287d65e52e74d5a3435569937adb075e3d" />
<Build Name="aspnet" BuildId="2.1.30-servicing-1293071+pb-20210811-01" ProductVersion="2.1.30" Branch="2.1" UniverseCommitHash="" />
<Build Name="cli215xx" BuildId="2.1.526" ProductVersion="2.1.526" Branch="release/2.1.5xx" Commit="26a950bee83e90bb453f5c78b40e9aac96ca5f1e" />
<Build Name="cli218xx" BuildId="2.1.818" ProductVersion="2.1.818" Branch="release/2.1.8xx" Commit="c98c51bb27cc65edb0eee9f28c0f4c2c4ddabaf2" />
```